3p/integration.js

```diff
@@ -138,9 +138,14 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
-const data = parseFragment(location.hash);
-window.context = data._context;
-
+const iframeName = window.name;
+let data;
+try {
+  data = JSON.parse(window.name).attributes;
```

Comment:
probably it's a good time to revisit the data structure, it's quite confusing now.

We can change it from
```
{
  ...
  attributes: {
    _context: {}
  }
}
```

to
{
  attributes: {},
  context: {},
}